code/Rounds/PreRound.cs

```diff
@@ -33,5 +45,20 @@ public override void OnPlayerSpawn(MainPlayer player)
             base.OnPlayerSpawn(player);
             AddPlayer(player);
         }
+
+        private List<Entities.SpawnPoint> GetSpawnPoints()
+        {
+            List<Entities.SpawnPoint> spawnPoints = new();
+            
+            foreach (Entity entity in Entity.All)
```

Comment:
Tu peux utiliser LINQ pour simplifier le foreach

Reply:
Ouaip j'ai vu, je savais pas si on le faisait

Reply:
Yeah we use it